docs/monitoring.md

```diff
@@ -1421,6 +1421,20 @@ Note: applies to the shuffle service
 - shuffle-server.usedDirectMemory
 - shuffle-server.usedHeapMemory
 
+- Notes: below shuffle service server-side metrics are specific to the Push-Based Shuffle
+(with `spark.shuffle.push.enabled` set as true on the client side, and with the server side
+flag `spark.shuffle.push.server.mergedShuffleFileManagerImpl` set as the appropriate
+org.apache.spark.network.shuffle.MergedShuffleFileManager implementation for the Push-Based
+Shuffle to be enabled)
```

Comment:
These metrics are not related to client side configurations, as if push is not enabled in client side, these metrics will still be shown up, but all 0.  Once server side configuration is set correctly to MergedShuffleFileManager, these metrics will be shown up.

```suggestion
- **notes:** The metrics below are specific to push-based shuffle and only emitted when 
  `spark.shuffle.push.server.mergedShuffleFileManagerImpl` is configured as 
  "org.apache.spark.network.shuffle.RemoteBlockPushResolver".
```
Please refer to other notes format in this file. You do need to follow the format there.

Reply:
I removed the client side config in the description. As for the format for note, I'm following the format at L1408, L1399, L1392, and L1384. This is the preview after the change:
![Screen Shot 2023-01-10 at 4 23 19 PM](https://user-images.githubusercontent.com/31781684/211689574-adf92b25-b0b7-406d-9dc4-847e64a09afd.png)
Let me know if you have further input, thanks @zhouyejoe !